dotnet/src/Functions/Functions.OpenAPI/Extensions/KernelOpenApiPluginExtensions.cs

```diff
@@ -268,9 +268,27 @@ async Task<RestApiOperationResponse> ExecuteAsync(SKContext context)
             })
             .ToList();
 
+        RestApiOperationResponse? restOperationResponse = null;
+        foreach (var response in operation.Responses)
+        {
+            if (response.Key == "200" || response.Key == "2XX" || (response.Key == "default" && restOperationResponse == null))
```

Comment:
nit: .StartsWith("2") ?

Reply:
Refactored this a bit. Feedback appreciated.